serverElectricUprights.js

```diff
@@ -0,0 +1,149 @@
+// using mLab/d2rd/d2rd-Notes db.collection(ElectricUprights)
+const express = require('express');
+const helmet = require('helmet');
+const mongoose = require('mongoose');
+// const Schema = mongoose.Schema;
+const cors = require('cors');
+const port = 5501;
+const server = express();
+
+// Define available models
+const ElectricUpright = require('./models/ElectricUpright')
+
+// define available databases
+const mLabD2rdNotes = 'mongodb://ds141611.mlab.com:41611/d2rd-notes';
+
+// Set active database
+const activeDB = mLabD2rdNotes; // avoids hardcoding db into mongoose.connect line 27
+
+// connect to database
+const options = {
+  user:"d2rd",
+  pass:"d2rd-PW",
+  useNewUrlParser: true // use the urlParser instead of the old one
+}
+// ☞ bd397750-457f-4308-b616-f0424ddc5d04
+
+mongoose.connect(activeDB, options)
```

Comment:
should only need one root server.js that spins up a server and connects to database.

Reply:
The multiple server files were experiments using different collections and db's as I was trying to figure out how to designate an activeDB.  This will go away once I clean up the code.